pkg/install/install.go

```diff
@@ -205,6 +205,33 @@ metadata:
 	return nil
 }
 
+func CamelK() error {
+	fmt.Println("🐪 Installing Apache Camel K  ... ")
+	registryAddress, err := getRegistryAddress()
+	if err != nil {
+		fmt.Printf("Error: %v\n", err)
+		return err
+	}
+
+	// Run the Helm install command
+	if err := runHelmInstall(registryAddress); err != nil {
+		fmt.Printf("Error: %v\n", err)
+		return err
+	}
+
+	if err := waitForCRDsEstablished(); err != nil {
+		return fmt.Errorf("crds: %w", err)
+	}
+	fmt.Println("    CRDs installed...")
+
+	if err := waitForPodsReady("default"); err != nil {
+		return fmt.Errorf("core: %w", err)
+	}
+	fmt.Println("    Apache Camel K installed...")
```

Comment:
should the camel-k operaror be using a different namespace?

Reply:
its their default, and the easiest. So for a quickstart I felt following this